Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -14,13 +14,25 @@ final class PledgeViewModelTests: TestCase {
   private let project = TestObserver<Project, Never>()
   private let reward = TestObserver<Reward, Never>()
   private let isLoggedIn = TestObserver<Bool, Never>()
+  private let total = TestObserver<Double, Never>()
+
+  private let reloadWithData = TestObserver<PledgeViewData, Never>()
+  private let updateWithData = TestObserver<PledgeViewData, Never>()
 
   override func setUp() {
     super.setUp()
 
+    self.vm.outputs.reloadWithData.observe(self.reloadWithData.observer)
     self.vm.outputs.reloadWithData.map { $0.0 }.observe(self.project.observer)
     self.vm.outputs.reloadWithData.map { $0.1 }.observe(self.reward.observer)
     self.vm.outputs.reloadWithData.map { $0.2 }.observe(self.isLoggedIn.observer)
+    self.vm.outputs.reloadWithData.map { $0.3 }.observe(self.total.observer)
+
+    self.vm.outputs.updateWithData.observe(self.updateWithData.observer)
+    self.vm.outputs.updateWithData.map { $0.0 }.observe(self.project.observer)
```

Comment:
Given that `reloadWithData` and `updateWithData` emit the same values and `reloadWithData` only emits once (and we test this), I figured it was fine to bind them to the same `TestObserver`s.